static/src/javascripts/projects/common/modules/commercial/commercial-feature-policies.js

```diff
@@ -68,16 +68,15 @@ define([
         }
     };
 
-    policies.membershipMessages = function () {
-        if (!detect.adblockInUseSync() &&
-            detect.getBreakpoint() !== 'mobile' &&
-            config.page.contentType === 'Article' &&
-            !userFeatures.isPayingMember()
-        ) {
-            return {
-                membershipMessages : true
-            };
-        }
+    policies.checkWeCanShowMembershipMessages = function () {
```

Comment:
the whole file is set up like that, and a `mapValues` creates a simple object out of them. 

i'd rather see this I think:
```javascript
policies.checkWeCanShowMembershipMessages = detect.adblockInUse.then(function (adBlockInUse) {
    return !adBlockInUse
        && …
});
```
but i don't really want to get in to refactoring commercial features as part of this :)

@jbreckmckye if you'd rather the name stayed the same, i have no problem with that, you touch this code far more than me anyway. shall i change it back?